polkadot/runtime/common/src/claims.rs

```diff
@@ -591,11 +591,9 @@ impl<T: Config> Pallet<T> {
 /// otherwise free to place on chain.
 #[derive(Encode, Decode, Clone, Eq, PartialEq, TypeInfo)]
 #[scale_info(skip_type_params(T))]
-pub struct PrevalidateAttests<T: Config + Send + Sync>(sp_std::marker::PhantomData<T>)
-where
-	<T as frame_system::Config>::RuntimeCall: IsSubType<Call<T>>;
+pub struct PrevalidateAttests<T>(core::marker::PhantomData<fn(T)>);
```

Comment:
Not sure we really need this :P

Reply:
WDYM?

Reply:
Yes

Reply:
But if i use `PhantomData<T>` then its not `Send + Sync` and if i remove the PD then it errors `parameter T is never used`.  
Signed Extension does require it though.

Reply:
Why does it needs to be Send and Sync?

Reply:
The `SignedExtension` trait [requires](https://github.com/paritytech/polkadot-sdk/blob/9d80735a7cccb7c52280ea057177d92e17203c51/substrate/primitives/runtime/src/traits.rs#L1477) it, i did not check why.  
The new TEs will also [require](https://github.com/paritytech/polkadot-sdk/blob/e8bbdeb6dd6a55455dc31669351cf8acd602f7c9/substrate/primitives/runtime/src/traits/transaction_extension/mod.rs#L48) it.